### PR TITLE
Add R package colorRamps.

### PR DESCRIPTION
**Pull request by jdblischak**
cc @bgruening

### CODE REVIEW

**Comment by jdblischak** [comment]
After 2 days of queuing, the Travis build was canceled. I just restarted it.

**Comment by jdblischak** [comment]
Hopefully the recent Travis updates will clear up the backlog:

> In an effort to clear up load on macOS servers, we are cancelling builds that have been waiting to start for more than 6 hours. This process will start at 16:15 CEST and is expected to take approximately 2 hours to complete.
>
> We will also be rolling back to a previous version of the worker to determine if recent changes have contributed to these issues. 

https://www.traviscistatus.com/incidents/h6gwvyq9zky0

File: recipes/r-colorramps/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    - r-base
+
+test:
+  commands:
+    - $R -e "library('colorRamps')"  # [not win]
+    - "\"%R%\" -e \"library('colorRamps')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=colorRamps
+  license: GPL
```
Comment (ocefpaf) [inline recipes/r-colorramps/meta.yaml]: Do you know the version?

Comment (jdblischak) [reply recipes/r-colorramps/meta.yaml]: The [DESCRIPTION file](https://cran.r-project.org/web/packages/colorRamps/DESCRIPTION) says only "GPL". The [CRAN entry](https://cran.r-project.org/package=colorRamps) expanded it to say:

> License: GPL-2 | GPL-3 [expanded from: GPL]

Would you prefer I put GPL2 or GPL3?

Comment (ocefpaf) [reply recipes/r-colorramps/meta.yaml]: That is up to the maintainers, conda-forge does not have a policy when there are alternatives.

**Comment by ocefpaf** [comment]
The license version can be addressed in the feedstock .Merging...

**Comment by jdblischak** [comment]
Unfortunately authors are allowed to submit to CRAN with the license designated simply as "GPL", so this particular package is not an anomaly. It's unclear to me how the conda-forge maintainers choosing a version could be legally binding when the original authors were vague as to their intentions (but I'm not a lawyer...).

Anyways, thanks for merging!

**Comment by ocefpaf** [comment]
>  It's unclear to me how the conda-forge maintainers choosing a version could be legally binding when the original authors were vague as to their intentions (but I'm not a lawyer...).

Same happens with PyPI :unamused:

**Comment by jdblischak** [comment]
> But you can list both and cover all bases

The license filed can have more than one entry? I didn't realize that. Should they be comma-separated?

```
license: GPL2, GPL3
```

What about `license_file`? Can that also have multiple entries? CRAN packages often have the author and year information in a file separate from the license text.

**Comment by ocefpaf** [comment]
I would do:

`license: GPL-2.0 and GPL-3.0`

> What about license_file?

There is not much we can do there.

**Comment by jdblischak** [comment]
> What about license_file?
>
> There is not much we can do there.

I'm confused. Why not? In your comment [here](https://github.com/conda-forge/staged-recipes/issues/3234#issuecomment-327781777), you suggested that we should include both the copy of the license and the license file included by the authors. If we can include multiple entries in license_file, that would be a solution, no?

**Comment by ocefpaf** [comment]
I meant we cannot do multiple license files yet as far as I know.
You can list multiple license, but add only a single file at the moment.

I concatenated files in the past, but that requires some extra work and I do not recommend start doing this everywhere. (It was a special case.)

**Comment by jdblischak** [comment]
> I meant we cannot do multiple license files yet as far as I know.
> You can list multiple license, but add only a single file at the moment.

OK. Makes sense.

> I concatenated files in the past, but that requires some extra work and I do not recommend start doing > this everywhere. (It was a special case.)

My thinking was to include the concatenation of the license files in the CRAN skeleton. Thoughts?

**Comment by ocefpaf** [comment]
> My thinking was to include the concatenation of the license files in the CRAN skeleton. Thoughts?

No strong opinion. If you are fine with that got for it!

**Pull request merged**